__mocks__/@grafana/runtime.ts

```diff
@@ -1,7 +1,9 @@
 import * as runtime from '@grafana/runtime';
 
 export const getBackendSrv = () => ({
-  datasourceRequest: jest.fn().mockImplementation(() => ({ ok: true })),
+  fetch: {
+    toPromise: () => jest.fn().mockImplementation(() => ({ ok: true })),
```

Comment:
**Nitpick:** maybe `jest.fn().mockResolvedValue({ ok: true })` as a bit of syntactic sugar ✨